tests/library/browsercontext-cookies.spec.ts

```diff
@@ -429,3 +429,37 @@ it('should parse cookie with large Max-Age correctly', async ({ server, page, de
     },
   ]);
 });
+
+it('iframe should inherit cookies from parent', { annotation: { type: 'issue', description: 'https://github.com/microsoft/playwright/issues/35439' } }, async ({ page, isLinux, browserName }) => {
+  await page.route('**', async (route, request) => {
```

Comment:
```suggestion
  await page.route('**/*', async (route, request) => {
```
nit: usually we do it like that.